new-lamassu-admin/src/components/editableTable/Row.js

```diff
@@ -132,7 +132,7 @@ const ECol = ({ editing, focus, config, extraPaddingRight, extraPadding }) => {
     SuffixComponent = TL2,
     textStyle = it => {},
     isHidden = it => false,
-    view = it => it?.toString(),
+    view = it => it?.toString() ?? '—',

```

Comment:
What is this for? Can't this logic be moved to the specific place where it's needed?